docs/advanced/backends.md

```diff
@@ -26,20 +30,23 @@ with MosaicBackend("https://mosaic.com/amosaic.json.gz") as mosaic:
 with MosaicBackend("dynamodb://us-east-1/amosaic") as mosaic:
     assert isinstance(mosaic, cogeo_mosaic.backends.dynamodb.DynamoDBBackend)
 
+with MosaicBackend("sqlite:///mosaic.db:amosaic") as mosaic:
+    assert isinstance(mosaic, cogeo_mosaic.backends.sqlite.SQLiteBackend)
+
 with MosaicBackend("file:///amosaic.json.gz") as mosaic:
     assert isinstance(mosaic, cogeo_mosaic.backends.file.FileBackend)
 
-# Create only
-with MosaicBackend("stac+https://my-stac.api/search", {"collections": ["satellite"]}, 10, 12) as mosaic:
-    assert isinstance(mosaic, cogeo_mosaic.backends.stac.STACBackend)
-
 with MosaicBackend("amosaic.json.gz") as mosaic:
     assert isinstance(mosaic, cogeo_mosaic.backends.file.FileBackend)
+
+# Read-Only
+with MosaicBackend("stac+https://my-stac.api/search", {"collections": ["satellite"]}, 10, 12) as mosaic:
+    assert isinstance(mosaic, cogeo_mosaic.backends.stac.STACBackend)
 ```
 
 ## STAC Backend
 
-The STACBackend is purely dynamic, meaning it's not used to read or write a file. This backend will POST to the input url looking for STAC items which will then be used to create the mosaicJSON.
+The STACBackend is a **read-only** backend, meaning it's not used to read or write a file. This backend will POST to the input url looking for STAC items which will then be used to create the mosaicJSON in memory.
```

Comment:
```suggestion
The STACBackend is a **read-only** backend, meaning it can't be used to write a file. This backend will POST to the input url looking for STAC items which will then be used to create the mosaicJSON in memory.
```